wiki/Glossary/en.md

```diff
@@ -712,7 +712,7 @@ Silences lasts for 4 weeks before being reset, along with the time that gets add
 
 ### Skin
 
-A set of graphics and custom sample sets (audio clips) that can replace the default ones used by osu!. This will allow for a more customised user experience. Most of what you see and hear in osu! can be skinned.
+A set of graphics and custom sample sets (audio clips) that can replace the default ones used by osu!. This will allow for a more customized user experience. Most of what you see and hear in osu! can be skinned.
```

Comment:
this is an alternative spelling
```suggestion
A set of graphics and custom sample sets (audio clips) that can replace the default ones used by osu!. This will allow for a more customised user experience. Most of what you see and hear in osu! can be skinned.
```